src/build.py

```diff
@@ -764,10 +764,15 @@ def OverrideCMakeCompiler():
         return []
     cc = 'clang-cl' if IsWindows() else 'clang'
     cxx = 'clang-cl' if IsWindows() else 'clang++'
-    return [
+    tools = [
         '-DCMAKE_C_COMPILER=' + Executable(GetPrebuiltClang(cc)),
-        '-DCMAKE_CXX_COMPILER=' + Executable(GetPrebuiltClang(cxx))
+        '-DCMAKE_CXX_COMPILER=' + Executable(GetPrebuiltClang(cxx)),
     ]
+    if IsWindows():
+        tools.append('-DCMAKE_LINKER=' +
+                     Executable(GetPrebuiltClang('lld-link')))
```

Comment:
why do we need this on windows specifically?

Reply:
On Windows, the linker is apparently invoked directly by CMake rather than indirectly by the compiler driver as on other platforms. So CMake itself needs to have the path to the linker.

Reply:
I see, thanks.